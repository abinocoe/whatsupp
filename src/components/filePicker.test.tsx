import * as React from "react"
import { shallow } from "enzyme"

import FilePicker from "./filePicker"

describe("FilePicker", () => {
  it("renders correctly", () => {
    const output = shallow(<FilePicker />)
    expect(output).toMatchSnapshot()
  })
})
