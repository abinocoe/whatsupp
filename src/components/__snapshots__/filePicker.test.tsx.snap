// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilePicker renders correctly 1`] = `ShallowWrapper {}`;
